project/sites/hiaredressni/themes/hiaredressni_theme/src/scss/components.scss

```diff
@@ -24,4 +24,16 @@
   }
 }
 
+.footer-area .column:last-child {
+  float: none;
+  width: unset;
+}
+
+#footer {
+  border-top: 9px solid $brand-primary;
```

Comment:
There is a variable in the init/_colors.scss which can set the footer border. It is OK to do this but may as well use the structure that is there. 
$footer-border: color(footer-border);
If you need a hand with this give me a message.

Reply:
Removed this code and updated colours.scss